### PR TITLE
Use getopt style short/long argument flags (without using getopt package)

### PR DESCRIPTION
**Pull request by wader**
Alternative to #30 without using getopt package. One difference is that multiple short flags can't be combined ex: `-ab` instead of `-a -b` does not work.

### CODE REVIEW

**Comment by wader** [comment]
@patrislav1 less code but works a little bit differently for short flags

**Comment by patrislav1** [comment]
don't have a strong opinion, I mean being able to combine flags is nice but it's not crucial and most of the flags require arguments anyway

File: README.md [hunk 1]
```diff
@@ -16,27 +16,23 @@ Produces [colortest.ansi.svg](cli/testdata/colortest.ansi.svg):
...
+--colorscheme       Color scheme
+--fontfile          Font file to use and embed
+--fontname          Font name
+--fontref           External font URL to use
+--fontsize          Font size
+--grid              Grid mode (sets position for each character)
+--help, -h          Show help
+--listcolorschemes  List color schemes
+--transparent       Transparent background
+--width, -w         Terminal width (auto if not set)
```
Comment (wader) [inline README.md:35]: I shorted some texts to stay within 80 characters. Otherwise help text look good? should include something more?

Comment (patrislav1) [reply README.md:35]: LGTM

**Comment by patrislav1** [comment]
> BTW if you want i can add you as a collaborator so you can do more things yourself, i trust your judgement :)

Thank you :smile: Yes it would make sense then I guess.

**Pull request merged**